pkg/controller/provider/web/ocp/client.go

```diff
@@ -114,6 +114,31 @@ func (c *Client) Path(object interface{}, id string) (path string, err error) {
 				Name:      name,
 			},
 		})
+	case *StorageClass:
+		if id == "" { // list
```

Comment:
is this conditional doing anything, since storageclasses don't have namespaces and `ns` doesn't seem to be used in the rest of the block?

Reply:
Ah, you're right.  cut-n-pasted and it's not needed.